src/soundFont2.ts

```diff
@@ -115,39 +115,37 @@ export class SoundFont2 {
     memoizedPresetNumber: number = 0
   ): Key | null {
     // Get a memoized version of the function
-    return memoize(
-      (keyNumber: number, bankNumber: number, presetNumber: number): Key | null => {
-        const bank = this.banks[bankNumber];
-        if (bank) {
-          const preset = bank.presets[presetNumber];
-          if (preset) {
-            const presetZone = preset.zones.find(zone => this.isKeyInRange(zone, keyNumber));
-            if (presetZone) {
-              const instrument = presetZone.instrument;
-              const instrumentZone = instrument.zones.find(zone =>
-                this.isKeyInRange(zone, keyNumber)
-              );
-              if (instrumentZone) {
-                const sample = instrumentZone.sample;
-                const generators = { ...presetZone.generators, ...instrumentZone.generators };
-                const modulators = { ...presetZone.modulators, ...instrumentZone.modulators };
+    return memoize((keyNumber: number, bankNumber: number, presetNumber: number): Key | null => {
```

Comment:
This change is just autoformatting.. it might have been caused by the husky hook